### PR TITLE
Add a set_media_path command to the ws server

### PR DESCRIPTION
**Pull request by geffrak**
JIRA: VMR-2452

DESCRIPTION

Add a set_media_path { "path": "/path/to/media" } command to the websocket
server so the websocket server can control what is shown in the TaskView's
player.

### CODE REVIEW

File: python/tk_desktop2/websockets/requests/sgc_actions/__init__.py [hunk 1]
```diff
@@ -7,4 +7,4 @@
 
 from .open_task import OpenTaskInSGCreateWebsocketsRequest
 from .open_task_board import OpenTaskBoardInSGCreateWebsocketsRequest
-
+from .set_media_path import SetMediaPathInSGCreateWebsocketsRequest
```
[the remark this answers is not in the document]
Comment (geffrak) [reply python/tk_desktop2/websockets/requests/sgc_actions/__init__.py]: It is...

[the remark this answers is not in the document]
Comment (jfboismenu) [reply python/tk_desktop2/websockets/requests/sgc_actions/__init__.py]: Not in this file it isn't! 😄

If you want to get rid of these errors, I recommend you put # nova at the end of the line.

File: python/tk_desktop2/websockets/requests/sgc_actions/set_media_path.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
+# Copyright 2018 Autodesk, Inc.  All rights reserved.
```
Comment (jfboismenu) [inline python/tk_desktop2/websockets/requests/sgc_actions/set_media_path.py]: 2019

File: python/tk_desktop2/websockets/requests/sgc_actions/set_media_path.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+import sgtk
+from ..request import WebsocketsRequest
+
+logger = sgtk.LogManager.get_logger(__name__)
+
+
+class SetMediaPathInSGCreateWebsocketsRequest(WebsocketsRequest):
+    """
+    Requests that set the current media in the Shotgun Create player.
```
Comment (jfboismenu) [inline python/tk_desktop2/websockets/requests/sgc_actions/set_media_path.py]: `Request that sets`

File: python/tk_desktop2/websockets/requests/sgc_actions/set_media_path.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+    This is a 'fire-and-forget' command that doesn't return anything.
+
+    Parameters dictionary is expected to be on the following form:
```
Comment (jfboismenu) [inline python/tk_desktop2/websockets/requests/sgc_actions/set_media_path.py]: `to be of the`

File: python/tk_desktop2/websockets/requests/sgc_actions/set_media_path.py [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+        :param connection: Associated :class:`WebsocketsConnection`.
+        :param int id: Id for this request.
+        :param dict params: Parameters payload from websockets.
+        """
+        super(SetMediaPathInSGCreateWebsocketsRequest,
+              self).__init__(connection, id)
+
+        self._bundle = sgtk.platform.current_bundle()
+
+        # validate
```
Comment (jfboismenu) [inline python/tk_desktop2/websockets/requests/sgc_actions/set_media_path.py]: I personally find that one word comment are rarely useful, you might as well remove it. The role if the if is quite clear.

**Pull request merged**